opencti-platform/opencti-front/src/private/components/cases/case_rfts/CaseRftKnowledgeGraphBar.jsx

```diff
@@ -306,6 +309,30 @@ class CaseRftKnowledgeGraphBar extends Component {
     this.handleCloseSelectByType();
   }
 
+  closeReferencesPopup() {
+    this.setState({ referenceDialogOpened: false });
+  }
+
+  submitReference(values, { setSubmitting, resetForm }) {
+    const { handleDeleteSelected } = this.props;
+    const { deleteObject } = this.state;
+    const commitMessage = values.message;
+    const references = R.pluck('value', values.references || []);
```

Comment:
```suggestion
    const references = (values.references || []).map(ref => ref.value);
```
Trying to not use ramda